packages/flutter_tools/test/integration.shard/android_workflow_integration_test.dart

```diff
@@ -0,0 +1,120 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/io.dart';
+import 'package:flutter_tools/src/convert.dart';
+import 'package:platform/platform.dart';
+import 'package:archive/archive.dart';
+import 'package:process/process.dart';
+
+import '../src/common.dart';
+
+const String kAndroidRepo = 'https://dl.google.com/android/repository/';
+
+const Map<String, String> kPlatformTools = <String, String>{
+  'windows': 'commandlinetools-win-6200805_latest.zip',
```

Comment:
not sure how stable the URL is .. it has a `latest` in there

Reply:
I believe that version number gets incremented from time to time, and that file may disappear if the version is old enough.